apps/ledger-live-desktop/.eslintrc.js

```diff
@@ -64,7 +66,7 @@ module.exports = {
         "no-prototype-builtins": 0,
         "promise/param-names": 0,
         "react-hooks/rules-of-hooks": "error", // Checks rules of Hooks
-        "react-hooks/exhaustive-deps": "warn", // Checks effect dependencies
+        "react-hooks/exhaustive-deps": "error", // Checks effect dependencies
```

Comment:
this change feels unrelated to what was necessary to change to reveal the problem and might be dangerous: sometimes there are reason NOT to add a given dependency because we don't want an effect to trigger in some change case (and instead the pattern becomes that we need to use a "ref")
i see that you introduced some new deps and this is a non trivial change of behavior that we need to carefully study case by case , which is a bit beyond the single LIVE-3599 topic

Reply:
Interesting! Thanks.